build_jdk11_image.sh

```diff
@@ -45,7 +45,7 @@ function build_docker_image {
 function main {
 	delete_local_images "${LIFERAY_DOCKER_REPOSITORY}/jdk11"
 
-	make_temp_directory templates/jdk11
+	make_temp_directory templates/jdk11 templates/jdk-common
```

Comment:
@Wykenz, can you please create a separated commit just to hold the creation and usage of the new folder `jdk-common`?